app/javascript/vue/shared/input.vue

```diff
@@ -42,9 +41,11 @@
       autoFocus: { type: Boolean, default: false },
       error: { type: String, required: false }
     },
-    watch: {
-      value: function(newVal) {
-        this.inputValue = newVal;
+    computed: {
+      inputValue() {
```

Comment:
could `<input type="number"` be used here in any way?

Reply:
I had difficulties handling decimal precisions if the input is of type number, so switched to the previous manage modal implementation using text type.